### PR TITLE
Database: Configure retention for tests and test_executions

### LINKED ISSUE #46: Database: Configure retention for tests and test_executions
Should be deleted after 30 days.

### PR DESCRIPTION
**Pull request by Cheshiriks**
### What's done:
  * Created mysql event scheduler

closes #46

### CODE REVIEW

File: db/changelog/v-1/event_scheduler.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="event-scheduler-1" author="frolov" context="prod or dev">
```
Comment (petertrr) [inline db/changelog/v-1/event_scheduler.xml:8]: I believe it should only be in prod profile, we don't use a persistent database in dev

File: db/changelog/v-1/event_scheduler.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <sql>
+            CREATE EVENT delete_test
+            ON SCHEDULE EVERY 1 DAY
+            STARTS TIMESTAMP(CURRENT_DATE)
```
Comment (petertrr) [inline db/changelog/v-1/event_scheduler.xml:12]: Will this be a time of script execution?

Comment (Cheshiriks) [reply db/changelog/v-1/event_scheduler.xml:12]: Yes it's

File: db/changelog/v-1/event_scheduler.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="event-scheduler-1" author="frolov" context="prod">
```
Comment (petertrr) [inline db/changelog/v-1/event_scheduler.xml:8]: We should think about a way to keep some sort of execution history even after we remove test executions, maybe store list of failed tests in Execution? Anyway, it's a thought for a separate issue

**Review by petertrr — APPROVED** [review]
lgtm

File: db/changelog/v-1/event_scheduler.xml (hunk 3 above)
Comment (Cheshiriks) [reply db/changelog/v-1/event_scheduler.xml:8]: Yes I agree. We need to create a issue for this.

Comment (petertrr) [reply db/changelog/v-1/event_scheduler.xml:8]: #69

**Pull request merged**